File: test_data/c_programs/gcc_testsuite/00102.c



extern int n;
extern void bar (void);
extern int baz (int);

void
foo (void)
{
  int i, j;
  for (j = 0; j < n; j++)
    {
      for (i = 1; i < j; i++)
 bar ();
      baz (0);
    }
}

File: test_data/java_programs/openjdk_testsuite/00102.java




public class NestedWildcards {

    public static void test(Box<String> b) {
        foo(bar(b));
    }
    private static <X> Box<? extends X> foo(Box<? extends X> ts) {
        return null;
    }
    public static <Y> Box<? extends Y> bar(Box<? extends Y> language) {
        return null;
    }

    interface Box<T> {}
}
